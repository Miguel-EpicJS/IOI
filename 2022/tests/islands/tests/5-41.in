4e5150f5a11b21d9ac3b44f2259e7a3452343ad4
20 9
0 7
0 19
8 0
15 8
19 6
17 15
11 17
7 11
6 19
