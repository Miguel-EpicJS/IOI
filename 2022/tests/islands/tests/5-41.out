91041670623a69783795ed82bda33426101ce311
OK
1
20
0 7 6 5 3 2 1 4 8 1 2 3 5 6 7 0 1 8 4 1
